plugins/module_utils/fusion.py

```diff
@@ -162,4 +166,7 @@ def fusion_argument_spec():
                 )
             ],
         ),
+        access_token=dict(
```

Comment:
Since we have 'constants' with the field names now, we could initialize this dict as
```
{
  PARAM_ISSUER_ID: ...
  PARAM_PRIVATE_KEY_FILE: ...
  PARAM_ACCESS_TOKEN: ... 
}
```
That way the constants are actually used and it helps avoiding typos (it's Python, so any help in that direction is welcome)